frontend/src/components/blog-feed.tsx

```diff
@@ -72,7 +72,7 @@ export default function BlogFeed() {
             <h2 className="mb-2 cursor-text text-xl font-semibold dark:text-dark-primary">
               Categories
             </h2>
-            <div className="flex flex-wrap gap-3 dark:rounded-lg dark:bg-dark-card dark:p-3">
+            <div className="flex flex-wrap gap-3">
               {categories.map((category) => (
```

Comment:
You have removed bg and rounded corners in darkmode?

Reply:
Yes , the issued raised highlighted the extra box on the tags under category section which caused a little miss alignment while toggling to dark mode so that box was removed so that the dark mode gets aligned with light mode

Reply:
@amit429 why did you have to remove dark:rounded-lg and dark:bg-dark ? the `dark:p-3` was the only guy giving problem, now try the dark mode this category card will never get to dark mode, because of your changes, 
Please test the changes before pushing, and if said politely take some time , understand what I am trying to convey and then get back. Randomly please don't reply back. It consumes lot of my time.

Reply:
hey i am sorry for not getting your point earlier. In the issue number #165 the guy mentioned : BUG- on left sidebar inside Categories Container there is a box showing, while change on Theme On Repetitive switch of Theme There can be seen under the "Categories Box", there is a box over "Tags".

I have comprehended it as it was the issue of the box getting created over the tags in dark mode which needed to be removed , so i had to remove the the background dark from the tags section under categories heading , i am confused on how the issue is just related to the dark:p-3

I have also attached the before after screenshots to show the difference , the issue suggested a bug regarding the dark box getting created over the tags. I have tested the code on local and attached the screenshot 

if i have comprehended it wrongly then please do let me know , i would like to revise the changes

Reply:
You are absolutely right @amit429, sorry for the rude behaviour and words. I was too focused on the alignment changes part, so thought p-4 is causing the problem, has forgotten this is the child of a container in which the dark theme background is already handled.

Thank you for the fix, Kindly make the changes in the PR description to have "Issues addressed section" and we are good to close.

Sincere apologies again for overlooking this. And thanks for your patience, much appreciated

Reply:
@krishnaacharyaa PR description have been updated and issues addressed section is added with reference to the screen shot. I would like to request you to please merge the pr.